src/index.ejs

```diff
@@ -17,7 +17,7 @@
     This CSP allows any SSL-enabled host, but you should limit these directives further to increase your app's security.
     Learn more about CSP policies at https://content-security-policy.com/#directive
   -->
-  <meta http-equiv="Content-Security-Policy" content="default-src 'self' https: localhost:*; script-src 'unsafe-inline' https: localhost:*; connect-src https: localhost:* ws://localhost:*; style-src 'unsafe-inline' https:; object-src 'none';">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self' https: localhost:*; script-src <% if (isLocal) { %>'unsafe-eval'<% }%> 'unsafe-inline' https: localhost:*; connect-src https: localhost:* ws://localhost:*; style-src 'unsafe-inline' https:; object-src 'none';">
```

Comment:
Something about the webpack config used for Vue 3 has introduced code that executes `'unsafe-eval'` in the browser, though only in development mode. No one has yet taken the time to delve into which plugin/loader/whatever in Webpack is the cause of this, so for now I'm allowing this in the CSP if running locally. Another option is to simply remove the CSP but having one is a best-practice that we encourage to increase your site's security.